test/headless/context.js

```diff
@@ -54,4 +59,59 @@ export default class Context {
     this._execution = this._execution.wait(time);
     return this;
   }
+
+  onDOMReady() {
+    this.wait('.tracked-item[data-id="1"]');
+    return this;
+  }
+
+  waitForImpression(identifierNum) {
```

Comment:
we do have default timeout of 200. `this.wait(`#impression-div-${identifierNum}`, **200**);`